src/components/App/components/UserInput/UserInput.js

```diff
@@ -68,15 +70,22 @@ function UserInput(props) {
                 inputProps={{
                   id: 'font-size',
                   type: 'number',
-                  min: MIN_FONT_SIZE,
-                  max: MAX_FONT_SIZE,
+                  min: MIN_FONT_SIZE_IN_PX,
+                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Yes, I thought about this a lot last night.

I don't like the idea of automatically adjusting the size input when you change the unit at all. It doesn't feel intuitive.

I would propose leaving it like this – I think the preview not updating when you get to those larger sizes is the lesser of the two evils.